lib/builders/genai-builder.ts

```diff
@@ -0,0 +1,54 @@
+import { BlueprintBuilder } from '../stacks';
+import * as utils from "../utils";
+import * as addons from '../addons';
+import * as spi from '../spi';
+import { NestedStack, NestedStackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class GenAIBuilder extends BlueprintBuilder {
```

Comment:
I am unclear on what specifically makes this builder a GenAI builder other than usage tracking? Is there anything that we should add specific to genAI like enable a particular instance types or deploy an example workload  - something?